bin/functions/load_config.py

```diff
@@ -185,12 +181,17 @@ def override_conf_by_paching_conf():
 def load_config(
         conf_root,
         workload_config_file,
-        workload_name,
+        workload_folder,
         patching_config=""):
     abspath = os.path.abspath
     conf_root = abspath(conf_root)
     workload_config_file = abspath(workload_config_file)
 
+    # get current workload job name and store it in workload_job
+    (dir,workload_job) = os.path.split(workload_folder)
```

Comment:
Yeah, workload_job is framework name such as spark, hadoop, prepare, etc.
Ok, I will rename it.